packages/playwright/tsconfig.json

```diff
@@ -67,7 +67,8 @@
     "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */,
 
     // Use ECMA-standard behavior for class properties.
-    "useDefineForClassFields": true
+    "useDefineForClassFields": true,
+    "skipLibCheck": true
```

Comment:
it's somewhat bizarre but the existing playwright types here (external package) have type issues in them so I had to disable `node_modules` checking here. I might investigate later if we can upgrade some versions to fix the issue but it isn't particularly important rn